owlbot.py

```diff
@@ -115,20 +137,16 @@ def update_fixup_scripts(library):
     cov_level=100,
 )
 
-s.move(
-    templated_files,
-)
-
-s.replace(
-    "noxfile.py",
-    "GOOGLE_APPLICATION_CREDENTIALS",
-    "FIRESTORE_APPLICATION_CREDENTIALS",
-)
+s.move(templated_files)
```

Comment:
The change to replace `GOOGLE_APPLICATION_CREDENTIALS` with `FIRESTORE_APPLICATION_CREDENTIALS` must be preserved:  firestore system tests rely on a different environment variable.

Reply:
```suggestion
s.move(templated_files)

s.replace(
    "noxfile.py",
    "GOOGLE_APPLICATION_CREDENTIALS",
    "FIRESTORE_APPLICATION_CREDENTIALS",
)
```

Reply:
I guess we've decided to drop the check for the environment variable in `noxfile.py` -- I'm not sure why, but it is gone in (some) other libraries, at least.